### PR TITLE
Migrate live bus feature

### PR DESCRIPTION
**Pull request by patrickbeck1337**
## Proposed Changes

  -

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform

closes #

### CODE REVIEW

**Comment by leonardehrenfried** [comment]
Again, can you please move the SVG to the source file?
----------------------
While talking to @aaronyamil yesterday I remembered something that I completely forgot! Last year I wrote a little "compilation" script that combines the upstream SVGs with our downstream ones without you having to manually copy over SVGs from one file to another.

It's documented here: https://github.com/stadtnavi/digitransit-ui/blob/next/docs/Icons.md

Can you please adjust your PR so that it takes advantage of this process?

I totally apologise for not remembering earlier.

File: app/configurations/realtimeUtils.js [hunk 1]
```diff
@@ -254,6 +254,19 @@ export default {
...
     routeSelector: defaultRouteSelector,
 
+    active: true,
+  },
+  hb: {
+    mqttTopicResolver: function mqttTopicResolver() {
+      return '/gtfsrt/vp/#';
+    },
+
+    mqtt: 'wss://api.staging.stadtnavi.eu:1884',
```
Comment (leonardehrenfried) [inline app/configurations/realtimeUtils.js:264]: The URL has changed and should be `wss://api.stadtnavi.de/mqtt`

**Comment by leonardehrenfried** [comment]
Please use the red/green bus icons on the map as well.

Screenshot from 2021-05-04 13-07-59

File: app/configurations/realtimeUtils.js (hunk 1 above)
Comment (patrickbeck1337) [reply app/configurations/realtimeUtils.js]: Done

**Comment by patrickbeck1337** [comment]
> Please use the red/green bus icons on the map as well.
> 
> Screenshot from 2021-05-04 13-07-59

Which variant should we use? Since they introduced new size classes for the icons there. 

This one is using the large icon class:
image


This one is using the small icon class:
image

**Comment by leonardehrenfried** [comment]
Definitely the small icon size.

Can you also make the text/icons in the second line a little larger? Also please line up the word "Bus" with "Niedrige Auslastung".

Lastly, CI reports unit test errors.

File: app/configurations/realtimeUtils.js (hunk 1 above)
Comment (leonardehrenfried) [reply app/configurations/realtimeUtils.js:264]: Is it? I still see `wss://api.staging.stadtnavi.eu:1884`.

Can you re-check?

Comment (patrickbeck1337) [reply app/configurations/realtimeUtils.js:264]: My bad, I reverted the change accidentally. Although the `wss://api.stadtnavi.de/mqtt` does not return any data yet.

Comment (leonardehrenfried) [reply app/configurations/realtimeUtils.js:264]: Hmm, maybe you need the trailing slash, so `wss://api.stadtnavi.de/mqtt`

**Comment by patrickbeck1337** [comment]
Ok I redesigned the component a bit. Recheck again. Looks like that now.

image

**Comment by leonardehrenfried** [comment]
Looks good, thanks!

Did you end up needing the trailing slash?

**Comment by leonardehrenfried** [comment]
I'm answering my own question: you need it. I pushed the change.

**Pull request merged**